isis/src/newhorizons/apps/leisa2isis/NewHorizonsLeisaArchive_fit.trn

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
A lot of these files had _fit at the end of the filename. Wasn't sure the significance of that is (if any), or whether or not I should have left it in. Feedback is appreciated